docs/ref/forms/api.txt

```diff
@@ -1416,7 +1439,7 @@ Methods of ``BoundField``
 
 .. method:: BoundField.render(template_name=None, context=None, renderer=None)
 
-    The render method is called by ``as_field_group``. All arguments are 
+    The render method is called by ``as_field_group``. All arguments are
```

Comment:
Despite this blank space deleted is a nice cleanup, it should not be part of this commit. It could be a previous commit or a different PR.

Reply:
I wonder whether those are not corrected by the pre-commit hooks.

Reply:
I don't understand exactly your question, but to be more precise: these empty space cleanups should not be part of this PR/commit, they should be a different commit or PR altogether. Thanks!

Reply:
since PyCharm is constantly autocorrecting this because of `trim_trailing_whitespace` = true in `.editorconfg` I can't prevent this from being part of the diff. I will always endup having this correction as soon as I open the file. It's just a pain to prevent. It does not seem to be worth the cost to leave this correction out of the diff so is it ok to leave it in?

Reply:
Thank you @christophehenry for the details. Sadly,  is not OK to leave those in, it's against policy. You are welcome to do a separated PR with those fixes, or a previous commit in this same PR. Ping me explicitly and I'll merge ASAP.

Reply:
[Done](https://github.com/django/django/pull/18856).